README.md

```diff
@@ -1,17 +1,26 @@
 # RuneWordFinder
```

Comment:
Put more official README in place since this is technically the first "official" merge.